websites/A/Alura/presence.ts

```diff
@@ -0,0 +1,278 @@
+			"UX & Design"
+		)
+			presenceData.largeImageKey = "ux";
+		else if (
+			document.querySelector("div.formacao__info-categoria > a").textContent ===
+			"Data Science"
+		)
+			presenceData.largeImageKey = "datascience";
+		else if (
+			document.querySelector("div.formacao__info-categoria > a").textContent ===
+			"Mobile"
+		)
+			presenceData.largeImageKey = "mobile";
+		else if (
+			document.querySelector("div.formacao__info-categoria > a").textContent ===
+			"Inovação & Gestão"
+		)
+			presenceData.largeImageKey = "inovacao";
+	} else if (document.location.pathname.includes("/courses"))
+		// My courses
+		presenceData.details = "Viewing my courses"; // alura.viewMycourses
+	// User page
+	else if (document.location.pathname.includes("/user")) {
+		if (document.querySelector(".profile-header-name") !== null) {
+			title = document.querySelector(".profile-header-name");
+			presenceData.details = strings.user; // general.viewUser
+			presenceData.state = title.textContent;
+		} else {
+			// Study Plans
+			if (document.location.pathname.includes("/planos-de-estudo"))
+				presenceData.details = "Viewing study plans"; // alura.viewStudyPlans
+			// Payments
+			else if (document.location.pathname.includes("/payments"))
+				presenceData.details = "Viewing payments"; // alura.viewPayments
+			// Ranking
+			else document.location.pathname.includes("/rank");
+			presenceData.details = "Viewing ranking"; // alura.viewRanking
+		}
+	} else if (document.location.pathname.includes("/aluraflix")) {
+		//Aluraflix
+		presenceData.largeImageKey = "aluraflix";
+		presenceData.details = "Browsing Aluraflix"; // alura.aluraflix
+	} else if (document.location.pathname.includes("/carreirasemfronteiras")) {
+		//Alura programs
+		presenceData.largeImageKey = "carreirasemfronteiras"; // I accidentally misspelled the name in Developer Portal
+		presenceData.details = 'Listening to "Carreiras sem Fronteiras"'; // alura.carreirassemfronteiras
+		if (
+			document.querySelector(".podcast-header-title") !== null &&
+			document.querySelector(".--margin-left-image") === null
+		) {
+			title = document.querySelector(".podcast-header-title");
+			presenceData.state = `"${title.textContent}"`;
+		}
+	} else if (document.location.pathname.includes("/cases")) {
+		//Alura Cases
+		presenceData.largeImageKey = "aluracases";
+		presenceData.details = "Watching a Case"; // alura.cases
+		if (
+			document.querySelector(".intro__title") !== null &&
+			document.location.pathname.includes("/extra")
+		) {
+			title = document.querySelector(".intro__title");
+			presenceData.state = `"${title.textContent}"`;
+		}
+	} else if (document.location.pathname.includes("/scubadev")) {
+		//ScubaDev
+		presenceData.largeImageKey = "scubadev";
+		presenceData.details = 'Listening to "ScubaDev"'; // alura.scubadev
+		if (
+			document.querySelector(".podcast-header-title") !== null &&
+			document.querySelector(".--margin-left-image") === null
```

Comment:
Update this on future references of `!== null` and `=== null`
```suggestion
			document.querySelector(".podcast-header-title") &&
			!document.querySelector(".--margin-left-image")
```